src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/Config/ServiceBusOptions.cs

```diff
@@ -4,13 +4,14 @@
 using System;
 using System.Threading.Tasks;
 using Microsoft.Azure.ServiceBus;
+using Microsoft.Azure.WebJobs.Hosting;
 
 namespace Microsoft.Azure.WebJobs.ServiceBus
 {
     /// <summary>
     /// Configuration options for the ServiceBus extension.
     /// </summary>
-    public class ServiceBusOptions
+    public class ServiceBusOptions : ILoggableOptions
```

Comment:
There are other option types that need to be logged, e.g. EventHub, Singleton, etc.